src/app/features/notification/notification.component.html

```diff
@@ -0,0 +1,89 @@
+<button
+  mat-icon-button
+  matTooltip="Notifications"
+  i18n-matTooltip="notifications toolbar icon tooltip"
+  [matMenuTriggerFor]="notificationList"
+>
+  <span
+    [matBadge]="
+      unreadNotifications?.length > 0 ? unreadNotifications?.length : null
+    "
+    matBadgeColor="accent"
+  >
+    <fa-icon class="white" icon="bell"></fa-icon>
+  </span>
+</button>
+
+<mat-menu #notificationList>
+  <div class="notification-panel">
+    <div
+      class="flex-row justify-space-between align-center notification-panel-header"
+    >
+      <h2 class="notification-title" i18n>Notifications</h2>
+
+      <button
+        mat-icon-button
+        [matMenuTriggerFor]="settingsMenu"
+        (click)="$event.stopPropagation()"
+        icon-button
+      >
+        <fa-icon icon="ellipsis"></fa-icon>
+      </button>
+    </div>
+
+    <!-- Tabs -->
+    <mat-tab-group
+      [(selectedIndex)]="selectedTab"
+      (click)="$event.stopPropagation()"
+    >
+      <mat-tab label="All" i18n-label></mat-tab>
+      <mat-tab label="Unread" i18n-label></mat-tab>
+    </mat-tab-group>
+
+    @if (selectedTab === 0 || selectedTab === 1) {
+      @for (
+        notification of selectedTab === 0
+          ? allNotifications
+          : unreadNotifications;
+        track notification
+      ) {
+        <app-notification-item
+          [notification]="notification"
+          (onDeleteNotification)="deleteNotification.bind(this, notification)"
+          (markAsRead)="selectedTab === 1 ? markAsRead(notification) : null"
+        ></app-notification-item>
+      }
+
+      @if (
+        (selectedTab === 0 ? allNotifications : unreadNotifications).length ===
+        0
+      ) {
+        <div class="flex-column no-notification-message">
+          <fa-icon
+            class="no-notification-icon"
+            [icon]="['far', 'bell']"
+          ></fa-icon>
+          <span class="no-notification-text" i18n
+            >You have no notifications</span
+          >
+        </div>
+      }
```

Comment:
we can probably use `@empty` here to save some code: https://angular.dev/api/core/@for